### DISCUSSION TITLE
Make alien artifact spawn inside artifact holder

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15394] [Ideas and suggestions]
Artifact holder can only spawn artifact when there's a artifact salvaging mission
```xml
<PreferredContainer primary="ruintreasure" amount="1" spawnprobability="0.05" />
<PreferredContainer secondary="ruinstoragelarge" amount="1" spawnprobability="0.01" />
```
 
![20241218162041](https://github.com/user-attachments/assets/a13c85b0-0cc9-4272-a86a-43c7a71f3418)

**Comment by Regalis11** [comment]
I don't think the provided XML is enough to implement this: we would also need to ensure that artifact missions can still spawn the artifact in the holders even if they happen to contain these randomly spawning ones.

**Comment by A2ur3** [comment]
v1.8.4.0

> Artifacts always spawn in artifact holders in ruins, even if you don't have an artifact mission active.
